harmonic/evidence.pyx

```diff
@@ -397,11 +397,19 @@ class Evidence:
 
         ln_x = self.ln_evidence_inv_var - 2.0 * self.ln_evidence_inv
         x = np.exp(ln_x)
-        ln_evidence = np.log( 1.0 + x ) - self.ln_evidence_inv
-        ln_evidence_std = 0.5*self.ln_evidence_inv_var \
+        self.ln_evidence = np.log( 1.0 + x ) - self.ln_evidence_inv
```

Comment:
These should not be made class attributes.  If they are then need to ensure they are always updated and correct.  At present a user could call `convert_log_space_var` without first having compute `ln_evidence` and `ln_evidence_std`.